addons/sale_pdf_quote_builder/static/src/js/quotation_document_kanban/quotation_document_kanban_widget.xml

```diff
@@ -7,6 +7,7 @@
     >
         <xpath expr="//div[hasclass('o_cp_buttons')]" position="inside">
             <UploadButton
+                t-if="formData.sale_order_template_id"
```

Comment:
It could be an acceptable fix in stable, but in master, I'd rather add an `invisible` attribute on the page (`addons/sale_pdf_quote_builder/views/sale_order_views.xml`) like in `sale`.

As not a lot of clients are in 18.0, I'd directly to this in 18.0, existing clients won't probably be annoyed by this bug and if they are support will just have to help them upgrade this view.

Reply:
I think, to be on the safe side, we can apply this fix in version 18.0 and hide the view in the master branch.